slices/python3.11-minimal.yaml

```diff
@@ -0,0 +1,13 @@
+package: python3.11-minimal
+
+slices:
+  bins:
+    essential:
+      - libc6_libs
```

Comment:
why is libc6 a dependency here? It is already a dependency of libexpat1 and zlib1g, so this is redundant